FPMUL/FPMUL_Driver.py

```diff
@@ -1,13 +1,56 @@
-from cocotb.drivers import Driver
+from cocotb.drivers import BusDriver, Driver
 from cocotb.decorators import coroutine
 
-class FPMUL_Driver(Driver):
-    def __init__(self, generator=None):
-        Driver.__init__(self)
+from IEEE754 import IEEE754
+from cocotb.binary import BinaryValue
+from cocotb.triggers import RisingEdge
+
+"""
+ntwong0
+We need the Driver to be based on BusDriver, so that we can drive a bus input dut
+We have a couple options
+1. One Driver driving all inputs to dut
+2. Multiple drivers driving separate inputs to dut
+
+Since factory options granularize the inputs of A and B, let's go with option 2.
+Expanding from this choice,
+1. input_driver_A     - FPMUL_OperandDriver
+2. input_driver_B     - FPMUL_OperandDriver
+3. input_driver_Start - BusDriver
+4. input_driver_Rst   - BusDriver
+
+<s>Wait. Then do we need an FPMUL_Driver? Let's try without.</s>
+NOPE. We need to define _driver_send()
+"""
+
+class FPMUL_OperandDriver(BusDriver):
+    def __init__(self, entity, name, clock, _signals, generator=None):

```

Comment:
can remove the `generator=None` parameter

Reply:
removed